site/static/docs/5.2/assets/brand/OBS-logo-formatted.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="305" height="110">
+  <g fill="none" fill-rule="evenodd">
+    <path fill="#FF7900" d="M0 0h110v110H0z"/>
```

Comment:
`<path></path>` ?

Reply:
We got in our documentation both versions so let's keep the original way it was created by the designers.